xbmc/music/ContextMenus.cpp

```diff
@@ -35,6 +36,22 @@
 namespace CONTEXTMENU
 {
 
+static bool IsMusicDbItem(const CFileItem& item)
+{
+  if (URIUtils::IsMusicDb(item.GetPath()))
```

Comment:
Result of a runtime test. 
All music nodes seem to have ``library:\\music\xxx` when I checked, so a check for `musicdb:\\` isn't helping us. We could do a new helper checking if the items are music library items.
There is a URIUtils class for checking library already, but it checks for both video, music and something else I've forgotten.
Se we could create `URIUtils::IsMusicLibrary`